Arduino/Arduino_Uno_R3/src/main.c

```diff
@@ -0,0 +1,12 @@
+#include "ArduinoUno.c"
+#define Arduino_h
+
+void setup () 
+{ 
+  pinMode(13, OUTPUT);
+}
+void loop ()
+{
+  GPIO_Write('A', 13, 1);

```

Comment:
You have passed a char instead of a char * @AndyPro720

Reply:
@rish292 char * isn't acting as a pointer but rather like a [string literal](https://stackoverflow.com/a/718482/12632622) here. So '' is valid here.

*Although for future definitions we will avoid that usage all together.

Reply:
![Screenshot 2023-04-30 at 3 38 45 PM](https://user-images.githubusercontent.com/104636130/235347547-2881be80-47ac-445f-a1c6-c084b37b6b34.png)

Reply:
![Screenshot 2023-04-30 at 3 40 55 PM](https://user-images.githubusercontent.com/104636130/235347635-793271df-a975-442c-bafd-1ca05d7a867a.png)

Reply:
?? @AndyPro720

Reply:
@rish292 It runs fine on my machine. Looking up few examples online, it seems to work fine on gcc but similar troubles have been reported with arduino compilation libs. So perhaps that.

Anyways, merge it for now. We'll iron it out by next PR.